cmd/entrypoint.go

```diff
@@ -42,11 +44,12 @@ func EntrypointCommand(cmd *cobra.Command, args []string) error {
 		execwrap.ExecWrap().RunSyscallExec(args)
 	} else {
 		err := execwrap.ExecWrap().RunCommandDirect(args)
-		if err != nil {
-			return err
-		}
-
-		err = CleanCommand(cmd, filesToDecrypt)
+		defer func(parentErr error) {
+			err := CleanCommand(cmd, filesToDecrypt)
```

Comment:
I recommend renaming to defferErr or something similar.  Since using`:=` controls if this is a new variable or shadows the function err I could see this easily being confused and a bug introduced later.